content/behind_the_scenes/index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Behind The Scenes"
+menu:
+  main:
+    weight: 1
+showDate: false
+draft: false
+---
+
+## Overview ##
+
+This article aims to explain how we have re-designed and re-written the backend used to collect, enrich and display the ADSB data our community sends us. 
+
+## Anatomy of a Feeder Connection ##
+When you follow our [docker_planewatch] 
+![plane.watch feeder connection anatomy](Connection%20Anatomy.drawio.png)
+
+1. [pw-feeder][pw-feeder] opens two connections to [Bordercontrol](#bordercontrol) - one connection for [BEAST][beast protocol], and another for [MLAT][mlat-client].
+2. During connection establishment, [Bordercontrol](#bordercontrol) checks the validity of the feeder's API key. If invalid, the connection is dropped. If valid, the connection is allowed, and [bordercontrol](#bordercontrol) starts a "feed-in" container for the client.
```

Comment:
This is a bit of duplication with the bordercontrol section below. Not sure if it should be slimmed down/simplified here and then expanded on below.  Probably to detailed here for the first overview section.